phpBB/phpbb/search/fulltext_mysql.php

```diff
@@ -78,12 +84,14 @@ class fulltext_mysql extends \phpbb\search\base
 	 * @param \phpbb\auth\auth $auth Auth object
 	 * @param \phpbb\config\config $config Config object
 	 * @param \phpbb\db\driver\driver_interface Database object
+	 * @param \phpbb\event\dispatcher_interface	$phpbb_dispatcher	Event dispatcher object
 	 * @param \phpbb\user $user User object
 	 */
-	public function __construct(&$error, $phpbb_root_path, $phpEx, $auth, $config, $db, $user)
+	public function __construct(&$error, $phpbb_root_path, $phpEx, $auth, $config, $db, $phpbb_dispatcher, $user)
```

Comment:
or does @Nicofuma have a good idea how to fix this? Maybe a factory, or make them services or whatever?